modules/services/kdeconnect.nix

```
@@ -48,7 +47,12 @@ in {
 
         Service = {
           Environment = "PATH=${config.home.profileDirectory}/bin";
-          ExecStart = "${cfg.package}/libexec/kdeconnectd";
+          ExecStart =
+            if strings.versionAtLeast (versions.majorMinor cfg.package.version)
+            "24.05" then
+              "${cfg.package}/bin/kdeconnectd"
+            else
+              "${cfg.package}/libexec/kdeconnectd";
```

I suggest simply doing

```suggestion
          ExecStart = "${cfg.package}/bin/kdeconnectd";
```

since it appears to me that version 24.05 has propagated to all the unstable channels, so no need to consider older versions.

---

Alright :)

---

Actually--I just remembered why I did it like that in the first place. The Qt5 version of KDE Connect is still available even in unstable *and* is still the default for this module in Home Manager, so if I did it like you suggested it'd actually leave it broken by default.

---

Fair enough, merged as-is 🙂